integrations/auth_ldap_test.go

```diff
@@ -208,6 +299,32 @@ func TestLDAPUserSigninFailed(t *testing.T) {
 	testLoginFailed(t, u.UserName, u.Password, i18n.Tr("en", "form.username_password_incorrect"))
 }
 
+func TestLDAPUserSigninFailedWithGroupsUsignDN(t *testing.T) {
```

Comment:
Minor typo `s/Usign/Using/g`
```suggestion
func TestLDAPUserSigninFailedWithGroupsUsingDN(t *testing.T) {
```